src/util/BarUtils.tsx

```diff
@@ -62,9 +62,9 @@ export const minPointSizeCallback =
   (minPointSize: MinPointSize, defaultValue = 0) =>
   (value: unknown, index: number): number => {
     if (typeof minPointSize === 'number') return minPointSize;
-    const isValueNumber = typeof value === 'number';
+    const isValueNumber = typeof value === 'number' || typeof value === 'undefined' || value === null;
```

Comment:
Can we just move the other two checks into the if statement? If not then change the variable name since it's no longer true that this is always a number

Reply:
No I mean line 66 - 

`isValueNumber || typeof value === ...etc` (sorry I'm on mobile)

Reply:
Also `value == null` would cover both null and undefined without needing the typeof check